src/02-elementare-strukturentheorie/halbgruppen.tex

```diff
@@ -0,0 +1,158 @@
+\section{Halbgruppen}
+
+\begin{theorem}[Fundamentalsatz der Arithmetik]
+
+\end{theorem}
+
+\begin{proof}
+    ..
+    \begin{itemize}
+        \notedate{29.03.2023}
+        \item Injektivität: Zu zeigen ist, dass es für alle $n \in \mathbb{N} \setminus \{0\}$ höchstens eine Primfaktorenzerlegung gibt. Wir wenden Induktion nach $n$ an:
+        
+        Induktionsanfang ($n=1$): Klarerweise hat $1$ nur die ``triviale'' Primfaktorenzerlegung, nämlich $0 \in S$, da jedes andere Produkt echt größer als $1$ ist.
+
+        Induktionsschritt ($k<n \implies n$): Sei indirekt angenommen $n$ hätte zwei Zerlegungen $n = p_1 \cdot ... \cdot p_e = q_1 \cdot ... \cdot q_m$, wobei $p_i, q_i \in \mathbb{P}$. Gibt es nun $i,j$ mit $p_i = q_j$, so betrachten wir
+        $$ \frac{n}{p_i} = p_1 \cdot ... \cdot p_{i-1} \cdot p_{i+1} \cdot ... \cdot p_\ell = q_1 \cdot ... \cdot q_{j-1} \cdot q_{j+1} \cdot ... \cdot q_m, $$
+        womit folgt, dass die Zerlegungen bereits gleich sind (bis auf Reihenfolge). Damit können wir von nun an annehmen, dass für alle $i, j$ $p_i \neq q_j$, \obda $p_1 < q_1$ gilt. Wir betrachten
```

Comment:
Damit können wir von nun an annehmen, dass ...
ab ... is der Satz recht schirch